vms/platformvm/state/camino.go

```diff
@@ -85,10 +85,10 @@ type CaminoDiff interface {
 	GetMultisigAlias(ids.ShortID) (*multisig.Alias, error)
 	SetMultisigAlias(*multisig.Alias)
 
-	// Consortium member nodes
+	// ShortIDsLink
 
-	SetNodeConsortiumMember(nodeID ids.NodeID, addr *ids.ShortID)
-	GetNodeConsortiumMember(nodeID ids.NodeID) (ids.ShortID, error)
+	SetShortLink(id ids.ShortID, key ShortLinkKey, link *ids.ShortID)
```

Comment:
```suggestion
	SetShortIDLink(id ids.ShortID, key ShortLinkKey, link *ids.ShortID)
```
just ShortLink as, well, short link, looks a bit confusing, imo
same for GetShortLink and modifiedShortLinks, shortLinksCache, shortLinksDB, ShortLinkKey, ShortLinkKeyRegisterNode, toShortLinkKey, fromShortLinkKey, writeShortLinks